app/plugins/rwahsNavigation/conf/rwahsNavigation.conf

```diff
@@ -0,0 +1,9 @@
+# RWAHS Navigation plugin configuration
+
+# Load plugin?
+enabled = 0
+
+# Custom search shortcuts hash.  Keys are used as navigation keys; each value has a `label`, `form_id` and `display_id`.
+# Both form_id and display_id are numeric values, the database ID of the `search_form` or `display` record respectively.
+advanced_search_shortcuts = {
```

Comment:
See https://github.com/rwahs/providence-configuration/pull/18 for the actual config.